src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,99 @@
+package wolox.training.models;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name="Users")
+public class User{
+	
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    
+    @NotNull
+    private String userName;
+    
+    @NotNull
+    private String name;
+    
+    @NotNull
+    private Date birthDate;
+    
+    @ManyToMany(cascade = {CascadeType.REFRESH, CascadeType.MERGE})
+//    @JoinTable(name = "user_book", 

```

Comment:
A good habit is to avoid leaving commented code in commits, it is preferable to delete them.

Reply:
Lo deje para tener en cuenta como hacer el cambio de nombre y el join, es para ayuda memoria!!

Reply:
Buenísimo no solo sabia, genial entonces no dije nada 👍